### PR TITLE
docs: readme

### PR DESCRIPTION
**Pull request by adlerhurst**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,138 +1,119 @@
...
+### Quick starts
 
-Customize login and console with your design  
-![private_labeling](https://user-images.githubusercontent.com/1366906/123089110-d148ff80-d426-11eb-9598-32b506f6d4fd.gif)
+To start developing your app as fast as possible check out our [quick start guides](https://docs.zitadel.ch/docs/quickstarts/introduction) where we describe basic examples in many languages.
 
+If your preferred language is missing please let us know [here](https://github.com/caos/zitadel/discussions/1717)
 
-## Other CAOS Projects
+<!-- TODO: add more quick starts, at least for the languages with client libraries -->
```
Comment (mffap) [inline README.md:102]: Would be nice if we could reference quick starts or examples per language

File: README.md [hunk 2]
```diff
@@ -1,138 +1,119 @@
...
 
-Register new applications
-![OIDC-Client-Register](https://user-images.githubusercontent.com/1366906/118765446-62064b80-b87b-11eb-8b24-4f4c365b8c58.gif)
+| Language | Client | ManagmentAPI | AdminAPI | AuthAPI | Thanks to the maintainers |
+|----------|--------|--------------|----------|---------|---------------------------|
+| .NET       | [zitadel-net](https://github.com/caos/zitadel-net) | ✔️ | ✔️ | ✔️ | [buehler 👑](https://github.com/buehler) |
+| Dart       | [zitadel-dart](https://github.com/caos/zitadel-dart) | ✔️ | ✔️ | ✔️ | [buehler 👑](https://github.com/buehler) |
+| Elixir     | [zitadel_api](https://github.com/jshmrtn/zitadel_api) | ✔️ | ✔️ | ✔️ | [jshmrtn 🙏🏻](https://github.com/jshmrtn) |
+| Go         | [zitadel-go](https://github.com/caos/zitadel-go) | ✔️ | ✔️ | ✔️ | ZITADEL |
+| Rust       | [zitadel-rust](https://crates.io/crates/zitadel) | ✔️ | ✔️ | ✔️ | [buehler 👑](https://github.com/buehler) |
```
Comment (mffap) [inline README.md:94]: Why not show important but missing languages? IMO would be useful to make others aware that this is something they can contribute in a meaningful way.

File: README.md [hunk 3]
```diff
@@ -1,138 +1,119 @@
...
+### Authentication
 
-* **ZITADEL Cloud:** [**ZITADEL.ch**](https://zitadel.ch) is our shared cloud service hosted in Switzerland. [Get started](https://docs.zitadel.ch/docs/guides/basics/get-started) and try the free tier, including already unlimited users and all necessary security features.
-* **ZITADEL Dedicated:** We operate and support a dedicated instance of **ZITADEL** for you. [Get in touch!](https://zitadel.ch/contact/)
+- Single Sign On (SSO)
+- Passwordless with FIDO2
+- Username / Password
+- Multifactor authentication with OTP, U2F
+- [Identity Brokering](https://docs.zitadel.ch/docs/guides/authentication/identity-brokering)
+- [Machine-to-machine (JWT profile)](https://docs.zitadel.ch/docs/guides/authentication/serviceusers)
```
Comment (mffap) [inline README.md:41]: PAT

File: README.md [hunk 4]
```diff
@@ -1,141 +1,145 @@
...
+[![Go Report Card](https://goreportcard.com/badge/github.com/caos/zitadel)](https://goreportcard.com/report/github.com/caos/zitadel)
+[![discord](https://badgen.net/discord/online-members/erh5Brh7jE)](https://zitadel.ch/chat)
+[![follow us](https://badgen.net/twitter/follow/zitadel)](https://twitter.com/zitadel)
+<a href="https://www.certification.openid.net/plan-detail.html?public=true&plan=w3ddtJcy0tpHL"><img src="./docs/static/logos/oidc-cert.png" alt="OpenID certification" height="35px" width="auto" /></a>
 
-We built **ZITADEL** not only with the vision of becoming a great open source project but also as a superb platform to support developers building their applications, without need to handle secure user login and account management themselves.
+---
 
-## How Does It Work
+The identity and access management (IAM) solution with the **strongest change track** built for modern **global scale** cloud capabilities. Thanks to it's unique approach to store user **activities as events**, every change of an object can be traced
```
Comment (mffap) [inline README.md:18]: @adlerhurst please align messaging with @fforootd and @juergrinaldi

see: https://github.com/caos/caos-internal/issues/255

File: README.md (hunk 2 above)
Comment (adlerhurst) [reply README.md]: added js, python, jvm

File: README.md (hunk 3 above)
Comment (adlerhurst) [reply README.md:41]: added

File: README.md (hunk 1 above)
Comment (adlerhurst) [reply README.md]: added per use case

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,46 +1,75 @@
...
-Checkout the following chapters to start ZITADEL locally.
+We strongly recomend to talk to us before you start contributing to streamline our and your work.
 
-### Build for local development
+We accept contributions through pull requests. You need a github account for that. If you are unfamiliar with git have a look at Github's documentation on [creating forks](https://help.github.com/articles/fork-a-repo) and [creating pull requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork). Please draft the pull request as soon as possible. Go through the following checklist before you submit the final pull request:
 
-* Check out our [development guide](guides/development.md).
+1. Create a feature branch from the `main`-branch
+2. [Merge](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging) the lastet commit of the `main`-branch
+3. <!-- TODO: additional steps or does the pipeline the rest? -->
```
Comment (adlerhurst) [inline CONTRIBUTING.md:40]: fix todo

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,46 +1,75 @@
...
+
+### Backend / Login
+
+To keep the code clean and understandable we use [golangci-lint](https://golangci-lint.run). We recommend to format the code with this linter while working on ZITADEL to simplify the review process. The configuration is locaed [here](./.golangci.yaml).
+
+To start the backend with a debugger run the [`main.go`-file](./main.go) located in the root of ZITADEL and provide the following arguments: <!-- TODO: args and flags-->. Ensure that the database is running by running `docker compose -f ./build/local/docker-compose.yml up database` <!-- TODO: validate command -->. For additional information please use the documentation of your IDE.
+
+### Console
+
+<!-- TODO: ask maxpe for infos -->
```
Comment (adlerhurst) [inline CONTRIBUTING.md:65]: @peintnermax

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -1,46 +1,75 @@
...
+
+### API Definitions
+
+Ensure the provided code meets the [offical style guide](https://developers.google.com/protocol-buffers/docs/style).
+
+<!-- TODO: docker command to generate stub-->
```
Comment (adlerhurst) [inline CONTRIBUTING.md:71]: @adlerhurst fix todo

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -1,46 +1,75 @@
...
+
+## Contribute Docs
+
+<!-- TODO: ask maxpe for infos -->
```
Comment (adlerhurst) [inline CONTRIBUTING.md:75]: @peintnermax

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -1,46 +1,75 @@
...
+| API definitions | Specifications of the API | [Protobuf](https://developers.google.com/protocol-buffers) | [./proto/zitadel](./proto/zitadel) |
 
-* Please read [Security Policy](SECURITY.md).
+Please validate and test the code before you contribute.
+
+### Backend / Login
+
+To keep the code clean and understandable we use [golangci-lint](https://golangci-lint.run). We recommend to format the code with this linter while working on ZITADEL to simplify the review process. The configuration is locaed [here](./.golangci.yaml).
+
+To start the backend with a debugger run the [`main.go`-file](./main.go) located in the root of ZITADEL and provide the following arguments: <!-- TODO: args and flags-->. Ensure that the database is running by running `docker compose -f ./build/local/docker-compose.yml up database` <!-- TODO: validate command -->. For additional information please use the documentation of your IDE.
```
Comment (adlerhurst) [inline CONTRIBUTING.md:61]: @adlerhurst fix todo

File: README.md [hunk 10]
```diff
@@ -1,141 +1,155 @@
...
-* [Say hello](https://zitadel.ch/contact/)
-* [Join our Discord channel](https://discord.gg/erh5Brh7jE)
+### Run with docker compose
 
-## How To Contribute
+Simply clone the repo and execute the following command in the cloned folder:
 
-Details about how to contribute you can find in the [Contribution Guide](CONTRIBUTING.md)
+```bash
+COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1 docker compose -f ./build/local/docker-compose.yml up
```
Comment (adlerhurst) [inline README.md:68]: @adlerhurst verify docker-compose

File: README.md [hunk 11]
```diff
@@ -1,141 +1,155 @@
...
-## Security
+### Run on your machine
 
-See the policy [here](./SECURITY.md)
+<!-- TODO: download/install zitadel  -->
 
-## Features of ZITADEL platform
+<!-- TODO: add basic start-with-init command -->
+<!-- TODO: insecure, should we write logs to a file? -->
+<!-- TODO: what will be printed? is it possible to open the browser and show the login screen? -->
```
Comment (adlerhurst) [inline README.md:77]: @adlerhurst fix todos

File: README.md [hunk 12]
```diff
@@ -1,141 +1,155 @@
...
+```
 
-Use our login widget to allow easy and sucure access to your applications and enjoy all the benefits of passwordless (FIDO 2 / WebAuthN):
+### Client libraries
 
-* works on all modern platforms, devices, and browsers
-* phishing resistant alternative
-* requires only one gesture by the user
-* easy [enrollment](https://docs.zitadel.ch/docs/manuals/user-factors) of the device during registration
+<!-- TODO: check other libraries -->
```
Comment (adlerhurst) [inline README.md:89]: @adlerhurst fix todo for elixir

File: README.md [hunk 13]
```diff
@@ -1,141 +1,155 @@
...
+| Go       | [zitadel-go](https://github.com/caos/zitadel-go) | GRPC | ✔️ | ✔️ | ZITADEL |
+| Rust     | [zitadel-rust](https://crates.io/crates/zitadel) | GRPC | ✔️ | | [buehler 👑](https://github.com/buehler) |
+| JVM      | ❌ | ❌ | ❌ | | Maybe you? |
+| Python   | ❌ | ❌ | ❌ | | Maybe you? |
+| Javascript | ❌ | ❌ | ❌ | | Maybe you? |
 
-### Admin Console
 
-Use [Console](https://docs.zitadel.ch/docs/manuals/introduction) or our [APIs](https://docs.zitadel.ch/docs/apis/introduction) to setup organizations, projects and applications.
+<!-- TODO: api grpc / rest -->
```
Comment (adlerhurst) [inline README.md:103]:
```suggestion
```

File: README.md [hunk 14]
```diff
@@ -1,141 +1,155 @@
...
+- [**OIDC for GO**](https://github.com/caos/oidc) - OpenID Connect certified SDK (client and server) for Go
+- [**ZITADEL Tools**](https://github.com/caos/zitadel-tools) - Go tool to convert  key file to privately signed JWT
+- [**ORBOS**](https://github.com/caos/orbos/) - GitOps everything
+
+## Usage data
+
+ZITADEL components send errors and usage data to CAOS Ltd., so that we are able to identify code improvement potential. If you don't want to send this data or don't have an internet connection, add the following lines to your custom configuration:
+
+```yaml
+TODO: add proper configuration
```
Comment (adlerhurst) [inline README.md:150]: @adlerhurst fix todo

File: build/local/docker-compose-dev.yml [hunk 15]
```diff
@@ -18,15 +18,12 @@ services:
...
-# Port 50002 serves the REST API
-# Port 50003 serves the login GUI
-  backend-run:
-    restart: on-failure
+  zitadel:
     networks:
       - zitadel
     healthcheck:
-      test: ["CMD", "curl", "-f", "http://localhost:50002/management/v1/healthz"]
+      # TODO: ask livio for health check
```
Comment (adlerhurst) [inline build/local/docker-compose-dev.yml:25]: @adlerhurst fix todo

File: build/local/docker-compose.yml [hunk 16]
```diff
@@ -0,0 +1,39 @@
...
+      start_period: 20s
+    ports:
+      - 8080:8080
+      - 26257:26257
+
+# starts the backend (API's) of ZITADEL
+  zitadel:
+    networks:
+      - zitadel
+      # TODO: correct image?
```
Comment (adlerhurst) [inline build/local/docker-compose.yml:24]: @adlerhurst fix todo

File: build/local/docker-compose.yml [hunk 17]
```diff
@@ -0,0 +1,39 @@
...
+    image: ghcr.io/caos/zitadel:latest
+    # TODO: define commands maybe additional command to init + start
```
Comment (adlerhurst) [inline build/local/docker-compose.yml:26]: @adlerhurst fix todo

File: build/local/docker-compose.yml [hunk 18]
```diff
@@ -0,0 +1,39 @@
...
+    command: init && zitadel start
+    healthcheck:
+      # TODO: ask livio for health check
```
Comment (adlerhurst) [inline build/local/docker-compose.yml:29]: @adlerhurst fix todo

**Review by adlerhurst — COMMENTED** [review]
commented on remaining todos

**Pull request closed without merging**